lib/bob/job/docker_checker.ex

```diff
@@ -13,7 +13,7 @@ defmodule Bob.Job.DockerChecker do
       "3.17.0"
     ],
     "ubuntu" => [
-      "jammy-20220428",
+      "jammy-20220101",
```

Comment:
This would be a downgrade.

Reply:
My bad. Done: fac7348b239bd570e6c626707a20900a7c7b4639